support/doc/tools.md

```diff
@@ -42,20 +41,12 @@ Clone the PeerTube repo to get the latest version (even if you are on your PeerT
 
 ```
 $ git clone https://github.com/Chocobozzz/PeerTube.git
-$ CLONE="$(pwd)/PeerTube"
+$ CLONE="$(pwd)/PeerTube && cd $CLONE"
 ```
 
-Run ``yarn install --pure-lockfile``
-```
-$ cd ${CLONE}
-$ yarn install --pure-lockfile
-```
+Run `yarn install --pure-lockfile` or `NOCLIENT=1 yarn install --pure-lockfile` if you wish to save space by not having dependencies for the web interface of PeerTube.
```

Comment:
I think it's better when users do not have a choice to make and when the command takes the full width so:

 ####

Install dependencies:

```
$ NOCLIENT=1 yarn install --pure-lockfile
```

Build server:

```
$ npm run build:server
```